src/source/Ice/IceAgent.c

```diff
@@ -1233,6 +1233,15 @@ STATUS iceCandidatePairCheckConnection(PStunPacket pStunBindingRequest, PIceAgen
 
     CHK(pStunAttributePriority != NULL, STATUS_INVALID_ARG);
 
+    if (pIceCandidatePair->local->ipAddress.family == KVS_IP_FAMILY_TYPE_IPV4) {
```

Comment:
lot of derefs happening here should we check `pIceCandidatePair->local` not NULL and `pIceCandidatePair->local->ipAddress` not NULL?

Reply:
Yeah I agree, let me add that.

Reply:
adding for `pIceCandidatePair->remote` and its `ipAddress` as well

Reply:
ipAddress is struct, so null checks are nonsensical-- added checks for local and remote.